### PR TITLE
DBZ-1358 Update to Kafka 2.3.0

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1358

### CODE REVIEW

File: debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java [hunk 1]
```diff
@@ -371,5 +372,21 @@ public void sleep(long ms) {
                 // Ignore
             }
         }
+
+        public void waitObject(Object obj, Supplier<Boolean> condition, long deadlineMs) throws InterruptedException {
```
Comment (gunnarmorling) [inline debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java]: Where's this used?

Comment (jpechane) [reply debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java]: @gunnarmorling It is an impleemntation of the new method added by the interface

Comment (gunnarmorling) [reply debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java]: Ok; so is there any reason for not simply using the existing `SystemTime` class? I cannot see any difference to our implementation.

Comment (jpechane) [reply debezium-core/src/test/java/io/debezium/kafka/KafkaServer.java]: @gunnarmorling Removed, probably present for historical reasons

**Comment by jpechane** [comment]
@gunnarmorling This is done, for some reasons Travis did not updated the status in Github but if you click on details of the build you'll see that it is completed and passed.

**Comment by gunnarmorling** [comment]
Thx, @jpechane!

**Pull request merged**